src/examples/stripe-bots/README.md

```diff
@@ -0,0 +1,56 @@
+# Stripe and Medplum
```

Comment:
Many users aren't tracking changes to the `medplum-demo-bots` repo. I would recommend that some or all of this content live in our main `medplum/docs` package, and link out to this bot.